test/optimizer/old_optimizer_test.cpp

```diff
@@ -1,6 +1,16 @@
-#include "common/harness.h"
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// old_optimizer_test.cpp
+//
+// Identification: test/optimizer/old_optimizer_test.cpp
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
 
-#define private public
+#include "common/harness.h"
```

Comment:
Not clear why harness.h added here. Unnecessary?

Reply:
Harness is needed for the tests unless I am mistaken, otherwise the test wont compile. I fixed the formatting though.